### PR TITLE
USS-1970 Add SlurmCluster update instructions

### PR DESCRIPTION
**Pull request by dgloe-hpe**
# Description

Add instructions for how to update a SlurmCluster after it's been created. Also, fix the capitalization of SlurmCluster and fix the heading level for some sections.

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Comment by mtupitsyn** [comment]
@dgloe-hpe In order to get this merged, please acquire sufficient number of approvals. I can give one, @nrockershousen could you please take a look?

**Review by mtupitsyn — APPROVED** [review]

File: operations/multi-tenancy/ExampleWorkflow.md [hunk 1]
```diff
@@ -242,12 +243,32 @@ All tenants can be booted and configured with a single CFS configuration that co
     slurmd_options: "--conf-server 10.156.124.108,10.156.124.109"
     ```
 
+## Updates
+
+After initial creation, the `SlurmCluster` resource may be updated with new
+settings. This is useful to correct errors with the initial deployment, or
+to update to new Slurm versions.
+
+1. (`ncn-mw#`) Edit the `SlurmCluster` file (example `devcls01a.yaml`)
```
Comment (nrockershousen) [inline operations/multi-tenancy/ExampleWorkflow.md]:
```suggestion
1. (`ncn-mw#`) Edit the `SlurmCluster` file (For example, `devcls01a.yaml`).
```

File: operations/multi-tenancy/ExampleWorkflow.md [hunk 2]
```diff
@@ -287,7 +308,7 @@ Filename:  `devten01a.yaml`
 
 Repeat this step as needed for additional tenants.
 
-### Create and apply `Slurmcluster` configuration file
+### Create and apply `SlurmCluster` configuration file
```
Comment (nrockershousen) [inline operations/multi-tenancy/ExampleWorkflow.md]:
```suggestion
### Create and apply the `SlurmCluster` configuration file
```

File: operations/multi-tenancy/ExampleWorkflow.md [hunk 3]
```diff
@@ -754,10 +775,10 @@ After CFS completes, login to either a tenant UAN (if available), or tenant Comp
               memory: 512Mi
     ```
 
-#### Appendix C - Slurm configuration
+### Appendix C - Slurm configuration
 
-First, you are responsible for divvying up the Slingshot VNI space among the primary `Slurmcluster` ('user' namespace) and any tenant `Slurmclusters`.
-Start with the primary `Slurmcluster`, and then configure each tenant.
+First, you are responsible for divvying up the Slingshot VNI space among the primary `SlurmCluster` ('user' namespace) and any tenant `SlurmClusters`.
```
Comment (nrockershousen) [inline operations/multi-tenancy/ExampleWorkflow.md]:
```suggestion
First, you are responsible for divvying up the HPE Slingshot VNI space among the primary `SlurmCluster` ('user' namespace) and any tenant `SlurmClusters`.
```

**Review by nrockershousen — APPROVED** [review]

**Comment by dgloe-hpe** [comment]
@mtupitsyn or @nrockershousen could one of you merge this for me? I don't have permission to.

**Review by spillerc-hpe — APPROVED** [review]

**Pull request merged**